CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+
+## Suggesting Changes
+
+Users are welcome to suggest code changes to Holoscan CLI in the form of [Issues](https://github.com/nvidia-holoscan/holoscan-cli/issues) or [Pull Requests](https://github.com/nvidia-holoscan/holoscan-cli/pulls).
+
+### Issues
+
+Please open a new [issue](https://github.com/nvidia-holoscan/holoscan-cli/issues) if you'd like to request a new feature
+or propose a feature design. You can tag a specific community maintainer in your post with "@", or we'll update
+when we've had a chance to review your post.
+
+### Pull Requests
+
+While we primarily develop Holoscan CLI internally, we also accept external contributions that help move the platform
+forward. We typically favor contributions that aim to fix an existing issue or improve documentation, but we'll also integrate new features
+and enhancements when they're aligned with the Holoscan CLI vision. Please check in with the development team to propose your idea before spending time working on new features. This will help prevent duplicate effort and avoid spending time on features that would be unlikely to be merged.
+
+You might contribute to Holoscan to help us address fixes earlier in our development cycle, or to suggest improvements to Holoscan CLI that you believe would broadly benefit the Holoscan community.
+
+Holoscan CLI follows a monthly release process that includes internal quality assurance. To add a fix or feature,
+we request that you [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) Holoscan CLI, develop in a branch, and submit your change as a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) against the latest Holoscan CLI release commit. If we accept your submission after external discussion, we will integrate those changes within our internal development and credit you in Git commit history. Any changes that we accept from community contributions will undergo quality assurance testing before they are included in the next Holoscan CLI release.
+
+**Note**: We recommend that new GitHub users read GitHub's [Getting Started](https://docs.github.com/en/get-started/start-your-journey) guide before opening their first pull request.
+
+## Tracking Development
+
+We take all community feedback into consideration. If we don't believe a proposed change aligns with our direction for Holoscan CLI, or if we don't expect we can prioritize a task within a reasonable time frame, we'll let you know by appropriately labeling or closing the issue or pull request with an explanatory comment.
+
+For items that we do plan to pursue or integrate, we use
+[GitHub Milestones](https://github.com/nvidia-holoscan/holoscan-cli/milestones)
+to communicate our release planning. We will add community issues and pull requests to the approximate monthly release
```

Comment:
To confirm, @mocsharp do you plan to maintain milestones for Holoscan CLI? If not we could remove this section

Reply:
Yes, I will be maintaining the milestones.